.github/workflows/build-and-run-smoke-tests.yml

```diff
@@ -3,23 +3,38 @@
 # | Edit _templates/dev.yml.erb instead.          |
 # +-----------------------------------------------+
 
-name: Dev
+name: Build and run smoke tests
 
 on:
   workflow_dispatch:
-  push:
+  workflow_run:
+    workflows: [Build Base Images]
+    types: [completed]
     branches-ignore:
     - master
 
 jobs:
+  on-base-images-build-success:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    steps:
+      - run: echo 'The triggering workflow passed'
+  on-base-images-build-failure:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'failure' }}
+    steps:
+      - run: |
+        echo 'The triggering workflow failed'
+        exit 1
```

Comment:
If the base image build fails, this will base itself off the early run's base images, and thus be unreliable.